src/components/ProgressCircular/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+import {
+    StyledProgressCircular,
+    StyledIconContainer,
+    StyledIcon,
+} from './styled';
+import RenderIf from '../RenderIf';
+import ProgressCircularSVG from './ProgressCircularSVG';
+import { BaseProps } from '../types';
+
+type Sizes = 'large' | 'medium' | 'small';
+type Variants = 'brand' | 'success' | 'warning' | 'error';
```

Comment:
Let's define this in only one place. You can export this here and not redeclare it in
```ProgressCircularSVG.tsx```